sugaroid/brain/hangman.py

```diff
@@ -222,6 +222,109 @@
 )[
     1:-1
 ]
+HANGAMN_MARVEL_WORDS="""
+captainamerica 
+thor 
+hawkeye 
+blackwidow 
+vision 
+thanos 
+tonystark 
+pepperpotts 
+happy 
+peterparker 
+auntmay 
+blackpanther 
+nickfury 
+hill 
+gamora 
+groot 
+rocketracoon 
+starlord 
+hulk 
+ultron 
+bucky 
+falcon 
+antman 
+wasp 
+wanda 
+jarvis 
+edith 
+mysterio 
+skull 
+drstrange
+vulture
+mj
+killmonger
+shuri
+nebula
+captainmarvel
+loki
+drax
+yondu
+valkyrie
+stanlee
+""".split(
+    "\n"
+)[
+    1:-1
+]
+
+HANGMAN_STICKFIGURE=["""
++---+
+    |   |
+        |
+        |
+        | 
+        |
+    =========""","""
+    +---+
+    |   |
+    0   |
+        |
+        |
+        |
+    =========""",""" 
+    +---+
+    |   |
+    0   |
+    |   |
+        |
+        |
+    ========= ""","""
+    +---+
+    |   |
+    0   |
+   /|   |
+        |
+        |
+    ========= ""","""
+    +---+
+    |   |
+    0   |
+   /|\  |
+        |
+        |
+    =========""","""
+    +---+
+    |   |
+    0   |
+   /|\  |
+   /    |
+        |
+    ========= ""","""
+    +---+
+    |   |
+    0   |
+   /|\  |
+   / \  |
+        |
+    ========= 
+"""].split(
+    "\n"
+)[
+    1:-1
```

Comment:
These lines are unnecessary. we cannot split a `list` too. So I guess we can remove the the lines from `].split.... 1:-1`

Reply:
ok